unit_tests/testIOutil/testIOutil.f90

```diff
@@ -109,9 +109,6 @@ SUBROUTINE testIO_Strings()
       string='testing'
       CALL strrep(string,'t','TT')
       ASSERT_EQ(TRIM(string),'TTesTTing','testing')
-      tmpStr='testing  '
-      CALL strrep(tmpStr,'t','TT')
-      ASSERT_EQ(TRIM(tmpStr),'TTesTTing','testing')
```

Comment:
Sure you want to completely delete this?  Seems like a reasonable test.

Reply:
Yeah probably not yet, I do forsee that being replaced entirely with StringType::replace though.

Reply:
Yeah, I meant more that instead of deleting it, the `strrep` call should be replaced with a `%replace` call.  It's a reasonable test, just on a soon to be defunct subroutine.

Reply:
Replace should be tested in strings.f90. When `strrep` goes away so should this.